irispy/new_sji.py

```diff
@@ -135,13 +124,13 @@ def read_iris_sji_level2_fits(filename):
     """
 
     # Open a fits file
-    my_file = fits.open(filename)
+    my_file = fits.open(filename, memmap=True, do_not_scale_image_data=True)
```

Comment:
The ```memmap``` kwarg also needs to be in the definition of the function so a user can turn it off an on.

I suggest the default should be ```False``` and then users can turn it on when they are dealing with dig data.  What do you think @tiagopereira ?

Reply:
memmap added to the definition of the function.
Default value of memmap turned to False